tpm_test.go

```diff
@@ -26,11 +27,16 @@ var ValidTPM = TPMDevice{
 }
 
 func TestTPMDevice(t *testing.T) {
+	chardevStr := "-chardev socket,id=chrtpm0,path=tpm.socket -tpmdev emulator,id=tpm0,chardev=chrtpm0 -device tpm-tis,tpmdev=tpm0"
+	if runtime.GOARCH == "aarch64" || runtime.GOARCH == "arm64" {
+		chardevStr = "-chardev socket,id=chrtpm0,path=tpm.socket -tpmdev emulator,id=tpm0,chardev=chrtpm0 -device tpm-tis-device,tpmdev=tpm0"
+	}
+		
```

Comment:
```suggestion
    chardevName := "tpm-tis"
	if runtime.GOARCH == "aarch64" || runtime.GOARCH == "arm64" {
		chardevName = "tpm-tis-device"
	}
	chardevStr := fmt.Sprintf("-chardev socket,id=chrtpm0,path=tpm.socket -tpmdev emulator,id=tpm0,chardev=chrtpm0 -device %s,tpmdev=tpm0", chardevName)
```

It's easier to see what part of the expected string is different without repeating the longer string in both cases.